pontoon/base/models.py

```diff
@@ -139,6 +147,11 @@ class Locale(models.Model):
     name = models.CharField(max_length=128)
     plural_rule = models.CharField(max_length=128, blank=True)
 
+    # Locale contains references to user groups who translate or manage them.
+    # Groups also store respective permissions for users.
+    translators_group = models.ForeignKey(Group, related_name='translated_locales')
```

Comment:
You might want to set `on_delete=models.SET_NULL` for these two attributes. Otherwise, if those groups are ever deleted, the entire locale will be deleted as well.